modules/multi-architecture-import-imagestreams.adoc

```diff
@@ -0,0 +1,72 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="multi-architecture-import-imagestreams_{context}"]
+
+= Importing manifest lists in image streams on your multi-architecture cluster 
+
+On a {product-title} {product-version} multi-architecture cluster, the image streams in the cluster do not import manifest lists automatically. You must manually change the default `importMode` option to the `PreserveOriginal` option in order to import the manifest list.
+
+[NOTE]
+====
+For this procedure to run successfully, the `referencePolicy.type` field of your `ImageStream` object must be set to the `Source` type. 
+[source,yaml]
+----
+referencePolicy:
+    type: Source 
+----
+====
+
+.Prerequisites 
+
+* You installed the Openshift CLI (`oc`).
+
+.Procedure 
+
+* The following example command shows how to patch the `ImageStream` cli-artifacts so that the `cli-artifacts:latest` image stream tag is imported as a manifest list.
```

Comment:
Just a suggestion for a small reword. Up to you if you want it.
```suggestion
* The following  command is an example for patching the `ImageStream` cli-artifacts so that the `cli-artifacts:latest` image stream tag is imported as a manifest list.
```